frontend/src/components/namespace/CreateNamespaceButton.tsx

```diff
@@ -143,15 +144,16 @@ export default function CreateNamespaceButton() {
         </DialogContent>
         <DialogActions>
           <Button
-            aria-label="Cancel"
+            data-testid="create-namespace-dialog-cancel-button"
             onClick={() => {
               setNamespaceDialogOpen(false);
             }}
           >
             {t('translation|Cancel')}
           </Button>
           <Button
-            aria-label="Create"
+            data-testid="create-namespace-dialog-create-button"
+            aria-label={t('translation|Create')}
```

Comment:
this aria-label also needs to be removed

Reply:
done!